test/system/170-run-userns.bats

```diff
@@ -36,6 +36,19 @@ function _require_crun() {
     is "$output" ".*457" "Check group leaked into container"
 }
 
+@test "rootful pod with custom ID mapping" {
+    skip_if_rootless "does not work rootless - rootful feature"
+    skip_if_remote "remote --uidmap is broken (see #14233)"
+    random_pod_name=$(random_string 30)
+    run_podman pod create --uidmap 0:200000:5000 --name=$random_pod_name
```

Comment:
There is no test here. This test passes on main, which means you're not actually testing anything. Could you please add something that actually tests your new functionality (and which fails when run against main)? And can you add a comment describing something like "prior to #14280 this would fail with blah-blah-error" ?